### PR TITLE
Feature patient query

### LINKED ISSUE #9: Patient schema and resolver.
Add a schema file following the dstu standard.

Create a resolver that returns a list of patients. (READ query)

### PR DESCRIPTION
**Pull request by ronniemara**
Resolves #9

### CODE REVIEW

File: resolvers.js [hunk 1]
```diff
@@ -67,7 +67,15 @@ const resolvers = {
 
       return questionnaireResponseItem.answer;
     }
-  }
-};
+	  //args: id of the patient
+	  //Using the patient id, find the correspoding patient
+    patient(root, args) {
+      return Fhir.getOne({ resource: 'Patient', id: args.id }).then(res => {
+	    console.log(res)
```
Comment (wichopy) [inline resolvers.js]: Can remove this.

File: resolvers.js [hunk 2]
```diff
@@ -67,7 +67,15 @@ const resolvers = {
...
+	    const result = res.entry.map(entry => entry.resource)
+	    return result;
+
+      }
+    };
```
Comment (wichopy) [inline resolvers.js]: The linter is showing some syntax errors, looks like this should be a big object.

screen shot 2019-01-18 at 11 43 32 pm

**Review by wichopy — CHANGES_REQUESTED** [review]
A few syntax errors to fix.

File: src/resolvers.js [hunk 3]
```diff
@@ -66,8 +66,16 @@ const resolvers = {
...
+	  //args: id of the patient
+	  //Using the patient id, find the correspoding patient
+    patient(root, args) {
+      return Fhir.getOne({ resource: 'Patient', id: args.id }).then(res => {
+	    console.log(res)
+	    const result = res.entry.map(entry => entry.resource)
+	    return result;
+
+      }
+    };
```
Comment (wichopy) [inline src/resolvers.js]: I think you are also missing an extra `}` here. Did you try running it locally?

**Comment by ronniemara** [comment]
@wichopy I added airbnb styleguide to this branch. The style guide is causing some errors and warnings during linting. Can you checkout this branch and fix the liniting errors and warnings?

**Comment by wichopy** [comment]
Added fixes. Should run now when you run `npm run start`

**Pull request merged**